sdk/communication/azure-communication-network-traversal/azure/communication/network/traversal/_version.py

```diff
@@ -0,0 +1,9 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+VERSION = "1.0.0b1 (2021-07-12)"
```

Comment:
This looks like incorrect version info. I don't think you need date here in version file.